msal/src/Microsoft.Identity.Client/Exceptions/MsalClientException.cs

```diff
@@ -129,6 +129,20 @@ public class MsalClientException : MsalException
         /// The library is loaded on a platform which is not supported.
         /// </summary>
         public const string PlatformNotSupported = "platform_not_supported";
+
+#if iOS
+        /// <summary>
+        /// Xamarin.iOS specific. This erorr indicates that keychain access has not be enabled for the application.
```

Comment:
Typo: erorr => error, otherwise excellent. Not sure we even want to mention the TeamId.
what about
From MSAL 2.x and ADAL 4.x, the keychain for the publisher need to be accessed in order to provide Single Sign On between applications of the same publisher.

the rest is perfect 👍 

note that the <CodesignEntitlements>Entitlements.plist</CodesignEntitlements> sentence will probably generate a build warning, I believe you need to use &amp;lt; for < and &amp;gt; for >

Reply:
great...thank you!

---
In reply to: [222516169](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1297#discussion_r222516169) [](ancestors = 222516169)